### PR TITLE
Buttons

### PR DESCRIPTION
**Pull request by szkieb**
added some buttons, needed for updated version of timer and task components

### CODE REVIEW

File: src/assets/ArrowDown.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+export function FeedSVG() {
```
Comment (NoHara42) [inline src/assets/ArrowDown.tsx]: "FeedSVG" is function name that has already been used.
You could use:
`import * as OtherSVG from "./ArrowDown`
But it's not ideal, make sure to make your SVG Components have unique function names.

File: src/components/buttons/AddTaskButton.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+export function AddTaskButton() {
+	const [formVisible, SetFormVisible] = useState(true);
+
+	let formOverlay;
+
+	// render the Form when FormVisible is set to true
+	useEffect(() => {
+		if (formVisible) {
+			formOverlay = (
```
Comment (NoHara42) [inline src/components/buttons/AddTaskButton.tsx:14]: This needs to be stateful to update the DOM when it is updated

File: src/components/buttons/custombutton.tsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+export function CustomButton({ text, clickHandler }: CustomButtonProps) {
+	return (
+		<button
+			onClick={clickHandler}
+			className="py-3 px-2 border border-blue-500 bg-slate-300 text-black"
+		>
+			{text}
+		</button>
+	);
+}
```
Comment (NoHara42) [inline src/components/buttons/custombutton.tsx:12]: Use consistent casing for the filename

File: src/index.css [hunk 4]
```diff
@@ -2,14 +2,12 @@
 @tailwind components;
 @tailwind utilities;
 
-/* We can use this selector to target active NavLinks */
-a.active {
-	@apply text-secondary;
-}
+/*
```
Comment (NoHara42) [inline src/index.css:6]: Remember to use base, components, utilities layers.
@layer base -> All of your HTML Element selectors
@layer components -> All of your component custom classes
@layer utilities -> For utility classes that you most likely only use once

**Review by NoHara42 — CHANGES_REQUESTED** [review]
@szkieb see comments

File: src/assets/ArrowDown.tsx (hunk 1 above)
Comment (szkieb) [reply src/assets/ArrowDown.tsx]: Names have been changed

File: src/components/buttons/custombutton.tsx (hunk 3 above)
Comment (szkieb) [reply src/components/buttons/custombutton.tsx:12]: Changed the file name

**Review by NoHara42 — APPROVED** [review]

**Pull request merged**